src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -256,12 +256,18 @@ impl Renderer for HtmlHandlebars {
 
         let theme = theme::Theme::new(book.get_theme_path());
 
-        debug!("[*]: Register handlebars template");
+        debug!("[*]: Register the index handlebars template");
         handlebars.register_template_string(
             "index",
             String::from_utf8(theme.index.clone())?,
         )?;
 
+        debug!("[*]: Register the header handlebars template");
+        handlebars.register_template_string(
```

Comment:
As far as I understand the [`register_partial`](https://docs.rs/handlebars/0.29.1/handlebars/struct.Handlebars.html#method.register_partial) would be contextually more appropriate (yes currently these two functions have identical implementation but this can change in the future)